cubeboot/source/settings.c

```diff
@@ -131,6 +131,15 @@ void load_settings() {
         settings.disable_mcp_select = disable_mcp_select;
     }
 
+    // suppress_boot_setuo_and_rtc_errors
+    int suppress_boot_setup_and_rtc_errors = 1;
+    if (!ini_sget(conf, "cubeboot", "suppress_boot_setup_and_rtc_errors", "%d", &suppress_boot_setup_and_rtc_errors)) {
```

Comment:
Looking at this again with fresh eyes, this does seem like a bit of a verbose ini setting, so I'm open to suggestions if you think it'd be better to keep it more concise (maybe just `suppress_boot_errors`?)